.gitignore

```diff
@@ -28,3 +28,6 @@ proxy/proxy
 
 # Vim binaries on mac
 *.swp
+
+# Local config
+local.json
```

Comment:
Why is this included in this pr? Not described in pr description

Reply:
It's from slackbot branch and I accidentally added the local.json to this branch

Reply:
Idiot